mysql-9.1.yaml

```diff
@@ -132,6 +132,93 @@ subpackages:
       runtime:
         - "${{package.name}}-oci-entrypoint"
 
+  - name: ${{package.name}}-bitnami-compat
+    dependencies:
+      provides:
+        - mysql-bitnami-compat=${{package.full-version}}
+      runtime:
+        - bash
+        - busybox
+        - coreutils
+        - ini-file
```

Comment:
I think somewhere down the road we'll move `ini-file` (it is using go toolchain as of now) to image level, not a blocker for now but we can start it today. :)

Reply:
Let's keep it as is for now. We can refactor it later, it's a minor change if we need it for any reason.